### PR TITLE
feat: add logging for CDN access token S3 interactions

### PR DESCRIPTION
**Pull request by n1ru4l**
### Background

Adds more verbose logging for identifying potential issues. (https://guild-oss.slack.com/archives/C05LPPEHZHN/p1698772873246899)

### Checklist

- [ ] Input validation
- [ ] Output encoding
- [ ] Authentication management
- [ ] Session management
- [ ] Access control
- [ ] Cryptographic practices
- [ ] Error handling and logging
- [ ] Data protection
- [ ] Communication security
- [ ] System configuration
- [ ] Database security
- [ ] File management
- [ ] Memory management
- [ ] Testing

### CODE REVIEW

File: packages/services/api/src/modules/cdn/providers/cdn.provider.ts [hunk 1]
```diff
@@ -122,9 +122,23 @@ export class CdnProvider {
     targetId: string;
     alias: string;
   }) {
+    this.logger.debug(
+      'Creating CDN Access Token. (targetId=%s, projectId=%s, targetId=%s)',
+      args.targetId,
+      args.projectId,
+      args.targetId,
```
Comment (kamilkisiela) [inline packages/services/api/src/modules/cdn/providers/cdn.provider.ts:129]: doubled targetId

Comment (kamilkisiela) [reply packages/services/api/src/modules/cdn/providers/cdn.provider.ts:129]: actually, in all the logs

File: packages/services/api/src/modules/cdn/providers/cdn.provider.ts [hunk 2]
```diff
@@ -154,12 +176,29 @@ export class CdnProvider {
...
 
     if (headResponse.status !== 404) {
+      this.logger.debug(
+        'Failed creating CDN access token. Head request on S3 returned unexpected status while checking token availability. ( targetId=%s, projectId=%s, targetId=%s, status=%s)',
+        args.targetId,
+        args.projectId,
+        args.targetId,
+        headResponse.status,
+      );
+      this.logger.debug(await headResponse.text());
```
[the remark this answers is not in the document]
Comment (kamilkisiela) [reply packages/services/api/src/modules/cdn/providers/cdn.provider.ts:186]: I think it's fine to do it on CF Worker but whatwg-node may scream a bit. We should either double check or clone.

[the remark this answers is not in the document]
Comment (n1ru4l) [reply packages/services/api/src/modules/cdn/providers/cdn.provider.ts:186]: Its not used anywhere else as the headResponse is only used in this method?

**Pull request merged**